docs/SetupTypes.md

```diff
@@ -38,11 +38,38 @@ const { remote } = require('webdriverio');
 
 Using WebdriverIO in standalone mode allows you to integrate this automation tool in your own (test) project to create a new automation library. Popular examples include [Chimp](https://chimp.readme.io) or [CodeceptJS](http://codecept.io). You can also write plain Node scripts to scrape the web for content (or anything else that requires a running browser).
 
+You can use the `@wdio/sync` package to transform all commands so they run synchronously. This especially simplifies your test as you don't have to deal with `async/await` anymore. Here is an example how you can run synchronous commands with WebdriverIO in a standalone script:
+
+```js
+// standalone.js
+const { remote } = require('webdriverio')
+const { runSync } = require('@wdio/sync')
+
+remote({
```

Comment:
I think that something like this would be more clear as to what is run but it might just be preference:
```
function runTestsSync(test) {
  return new Promise((resolve, reject) => {
    return runSync(test)(resolve, reject)
  })
}

remote({
    runner: true,
    outputDir: __dirname,
    capabilities: {
        browserName: 'chrome'
    }
}).then((browser) => runTestsSync(() => {
    browser.url('https://webdriver.io')
    console.log(browser.getTitle())
    browser.deleteSession()
}))
```

Reply:
we could export `runTestsSync` to have it like:

```js
import sync from '@wdio/sync'

remote({
    runner: true,
    outputDir: __dirname,
    capabilities: {
        browserName: 'chrome'
    }
}).then((browser) => sync(() => {
    browser.url('https://webdriver.io')
    console.log(browser.getTitle())
    browser.deleteSession()
}))
```